internal/pkg/template/templates/environment/cf.yml

```
@@ -189,6 +189,24 @@ Resources:
       Tags:
         - Key: Name
           Value: !Sub 'copilot-${AppName}-${EnvironmentName}-env'
+{{- if (gt (len .VPCConfig.SecurityGroupConfig.Ingress) 0)}}
+      SecurityGroupIngress:
+      {{range $ind, $securityRule := .VPCConfig.SecurityGroupConfig.Ingress}}
```

If we don't need to use `$idx` we could also:
```suggestion
      {{range $securityRule := .VPCConfig.SecurityGroupConfig.Ingress}}
```